.github/workflows/build-docs.yml

```diff
@@ -81,13 +82,24 @@ jobs:
             uses: actions/checkout@v4
             with:
                 ref: ${{ github.event.pull_request.head.sha }}
+        -   name: Check if we are doing a full or partial build
```

Comment:
this 'pattern' appears now more than 3 times in this workflow and should be refactored (in a follow up PR)